### PR TITLE
Rename Topics to Thematic Areas

### PR DESCRIPTION
**Pull request by georgiana-b**
Together with https://github.com/NextGeoss/ckanext-nextgeoss/pull/103 solves https://github.com/NextGeoss/nextgeoss-catalogue-issues/issues/30.

Also added some documentation about the group parameters used in this plugin.

### CODE REVIEW

File: ckanext/grouphierarchy/templates/group/about.html [hunk 1]
```diff
@@ -7,17 +7,17 @@
     {{ super() }}
 
     {% block package_description %}
+      {% set name = c.group_dict.title %}
```
Comment (JGulic) [inline ckanext/grouphierarchy/templates/group/about.html]: setting the name variable inside the block causes the Title not to be shown on the Topic about page. Move it on the top, where the group variable is set.

**Review by JGulic — CHANGES_REQUESTED** [review]
I've only pulled the changes from this extension locally and the title of the Topic was not shown in the about page. With the other changes on ckanext-nextgeoss it works, and the title is shown.
But let's move the name variable on top of the page, just so we have all the changes that affect this extension in it.

**Comment by georgiana-b** [comment]
@JGulic Moved `name` at the top of the template as requested. Feel free to merge if you think it's ready.

**Review by JGulic — APPROVED** [review]
ACK

**Pull request merged**